src/worker/index.ts

```
@@ -0,0 +1,303 @@
+      stateMachineRepo ??
+      new StateMachineRepoMongo({
+        mongoClient: new MongoClient(config.get<string>('mongo.url'), {
+          // TODO: Do we need any options here?
+          // useNewUrlParser: true,
+          // useUnifiedTopology: true,
+        }),
+        logger: this.#logger,
+      });
+
+    // TODO: Start processing
+    this.#delay(this.#queueInterval).then(() => this.#processMessages());
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)